frontend/simple/js/router.js

```diff
@@ -0,0 +1,160 @@
+import Router from 'vue-router'
```

Comment:
Moving this to its own file was a great idea, good job! `main.js` is much nicer now! 😄 👍